flink-table/flink-table-api-scala-bridge/src/main/scala/org/apache/flink/table/api/scala/StreamTableEnvironment.scala

```diff
@@ -247,40 +247,50 @@ object StreamTableEnvironment {
     * The [[TableEnvironment]] for a Scala [[StreamExecutionEnvironment]] that works with
     * [[DataStream]]s.
     *
-    * A TableEnvironment can be used to:
-    * - convert a [[DataStream]] to a [[Table]]
-    * - register a [[DataStream]] in the [[TableEnvironment]]'s catalog
-    * - register a [[Table]] in the [[TableEnvironment]]'s catalog
-    * - scan a registered table to obtain a [[Table]]
-    * - specify a SQL query on registered tables to obtain a [[Table]]
-    * - convert a [[Table]] into a [[DataStream]]
-    * - explain the AST and execution plan of a [[Table]]
-    *
     * @param executionEnvironment The Scala [[StreamExecutionEnvironment]] of the TableEnvironment.
     */
   def create(executionEnvironment: StreamExecutionEnvironment): StreamTableEnvironment = {
-    create(executionEnvironment, new TableConfig)
+    create(
+      executionEnvironment,
+      EnvironmentSettings.newInstance()
+        .useAnyPlanner()
+        .inStreamMode()
+        .build())
   }
 
   /**
     * The [[TableEnvironment]] for a Scala [[StreamExecutionEnvironment]] that works with
     * [[DataStream]]s.
     *
-    * A TableEnvironment can be used to:
```

Comment:
Why did you remove this documentation?

Reply:
It's duplicated on the class level, but I'm ok with reverting that.

Reply:
I reverted that